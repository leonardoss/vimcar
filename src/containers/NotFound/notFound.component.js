import React from "react";
import Layout from "../../layouts";

const NotFound = () => (
  <Layout>
    <p>Not found</p>
  </Layout>
);

export default NotFound;
