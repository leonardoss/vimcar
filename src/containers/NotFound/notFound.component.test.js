import React from "react";
import Adapter from "enzyme-adapter-react-16";
import { configure, shallow } from "enzyme";
import { HashRouter as Router } from "react-router-dom";

import NotFound from "./";

configure({ adapter: new Adapter() });

describe("Testing <NotFound container", () => {
  it("should render", () => {
    expect(tree()).toHaveLength(1);
  });
  it("should match the snapshot", () => {
    expect(tree()).toMatchSnapshot();
  });
});

const tree = () =>
  shallow(
    <Router>
      <NotFound />
    </Router>
  );
