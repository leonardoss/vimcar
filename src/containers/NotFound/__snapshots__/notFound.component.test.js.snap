// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <NotFound container should match the snapshot 1`] = `ShallowWrapper {}`;
